source/whats-new.txt

```diff
@@ -12,11 +12,23 @@ What's New
 
 Learn what's new in:
 
+* :ref:`Version 4.3 <version-4.3>`
 * :ref:`Version 4.2 <version-4.2>`
 * :ref:`Version 4.1 <version-4.1>`
 * :ref:`Version 4.0 <version-4.0>`
 * :ref:`Version 3.6 <version-3.6>`
 
+.. _version-4.3:
+
+What's New in 4.3
+-----------------
+
+New features of the 4.3 Node.js driver release include:
+
+- SOCKS5 support
+- Option to disable UTF-8 validation
+- Type inference for nested documents
```

Comment:
Question: Is this related to [DOCSP-20071](https://jira.mongodb.org/browse/DOCSP-20071)? If yes, maybe the wording of this could be changed to stress dot notation. Maybe "supporting type inference in queries written using [dot notation](https://docs.mongodb.com/manual/core/document/#dot-notation)"

Reply:
This is directly from the release notes. I would expect this to be updated once those tickets are completed. In this case, the tickets were not completed before release, so there's no content to link to in order to clarify.